anomaly_detector/events/__init__.py

```diff
@@ -1 +1,4 @@
 """Event package."""
+from anomaly_detector.events.anomaly_event import AnomalyEvent
+
+__all__ = ['AnomalyEvent']
```

Comment:
Already updated in #225, please rebase - thanks.

Reply:
That is correct. This was a cherry pick of the commits from that PR. I'm going to rebase so changes should be good now.